ts/boot/configureStoreAndPersistor.ts

```diff
@@ -400,6 +402,22 @@ function configureStoreAndPersistor(): { store: Store; persistor: Persistor } {
   // Run the main saga
   sagaMiddleware.run(rootSaga);
 
+  // Handle migration of non-paginated messages
+  if (usePaginatedMessages) {
+    migrateToPagination(store, _ => Promise.resolve())
```

Comment:
What do you think about replacing the `Promise.resolve()` to `Promise.reject()`? This is just to prevent the very remote case in which `usePaginatedMessages` could be set to `true` with this stub still here, and this would cause all the messages to be removed, isn't it?

Reply:
Great point! Agree 100% [here it is](https://github.com/pagopa/io-app/pull/3841/commits/de7d277df978447b6a2ee4c0702cd5747bbc5b94).